benchmarks/cpp/nvfuser/layer_norm.cpp

```diff
@@ -43,13 +43,19 @@ static void setupLayerNorm(Fusion* fusion, DataType dtype) {
 
   auto layer_norm_results = layer_norm(input, 1, weight, bias, eps_ptr);
 
-  auto output = layer_norm_results.output;
+  auto output        = layer_norm_results.output;
+  auto output_mean   = layer_norm_results.mean;
+  auto output_invstd = layer_norm_results.invstd;
 
   if (dtype != DataType::Float) {
-    output = castOp(dtype, output);
+    output        = castOp(dtype, output);
+    output_mean   = castOp(dtype, output_mean);
```

Comment:
Curious why you changed here.

Reply:
To make the compare is apple-to-apple as the aten vectorized_layer_norm_kernel has three outputs.

Reply:
Ah, OK. Looks like the other outputs are not half in the aten implementation.

Did you see any perf difference with the additional outputs?

Reply:
changed them back to float. No noticible influence on performance.